content/posts/numpy/numpy-rng/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Good practices with numpy random number generators"
```

Comment:
When referring to the name of the project (so not code), we should use the proper capitalisation.
```suggestion
title: "Good practices with NumPy random number generators"
```